src/amo/actions/reviews.js

```diff
@@ -1,15 +1,18 @@
 /* @flow */
 import {
-  SHOW_EDIT_REVIEW_FORM,
-  SHOW_REPLY_TO_REVIEW_FORM,
   FETCH_REVIEWS,
   HIDE_EDIT_REVIEW_FORM,
   HIDE_REPLY_TO_REVIEW_FORM,
   SEND_REPLY_TO_REVIEW,
+  SEND_REVIEW_FLAG,
```

Comment:
I don't like the word "flag" as it's vague and a bit unclear. For the other thing we used `SEND_ABUSE_REPORT` so I'd say `REPORT_REVIEW_FOR_ABUSE` or `SEND_REVIEW_ABUSE_REPORT`?